packages/amplify_authenticator/lib/src/l10n/country_resolver.dart

```diff
@@ -20,1722 +22,1492 @@ import 'package:flutter/material.dart';
 enum CountryResolverKey {
   selectDialCode,
   noDialCodeSearchResults,
-
```

Comment:
Please run `flutter format .` from the authenticator package.

Reply:
@dnys1 I updated the l10n generate script to generate country localizations, and then run `flutter format` on the generated files. Let me know what you think.